qiskit/backends/_qeremote.py

```diff
@@ -0,0 +1,112 @@
+import time
+import logging
+from qiskit.backends._basebackend import BaseBackend
+from qiskit.backends._backendutils import get_backend_configuration
+from qiskit import _openquantumcompiler as openquantumcompiler
+from qiskit._result import Result
+from IBMQuantumExperience.IBMQuantumExperience import IBMQuantumExperience
+
+logger = logging.getLogger(__name__)
+
+class QeRemote(BaseBackend):
+    def __init__(self, configuration=None):
+        """Initialize remote backend for IBM Quantum Experience.
+
+        Args:
+
+        """
+        if configuration is None:
+            self._configuration = get_backend_configuration(backend_name)
```

Comment:
It seems `backend_name` is undefined when this point is reached - would it be possible to revise it?

Reply:
Yes